### DISCUSSION TITLE
安装nvm桌面下载vue-cli脚手架，但是并不能直接全局使用这是为啥

### DISCUSSION THREAD

**Discussion by kingyzx** [1111mp/nvm-desktop#100] [Q&A]
安装nvm桌面下载vue-cli脚手架，但是并不能直接全局使用这是为啥

**Comment by 1111mp** [comment]
你本地通过什么命令安装的，`npm install @vue/cli -g` 吗？ 能截图看一下具体报什么错吗？

你本地`nvm-desktop`运行都正常吗 比如切换node版本这些 还有使用`nvmd --help`这些

或者你是否通过 `npm config set prefix "/path/to/folder"` 更改过npm全局包的安装位置 可以通过`npm config get prefix`查看验证 如果更改过需要将 `"/path/to/folder"` 添加到系统环境变量里去 更多详情你可以查看这里：https://github.com/1111mp/nvm-desktop/issues/23
